docs/tutorials/misc/300_otcp.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>This cell should belong in a new section that is related to model/OTCP fitting and calibration (as opposed to the previous section - data preparation). Please add a new header above.</p><p>Also, we should explain some of the most important arguments for <code>OTCP</code> , especially during <code>fit_transport</code>:<code>epsilon </code> and <code>n_target</code></p><p>Lastly, we should describe what happens in <code>fit_transport</code> a bit as well (solving OT problem, etc.). </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>